src/color.rs

```diff
@@ -416,6 +555,36 @@ mod tests {
         assert!(!Color {red: 0.0, green: 0.0, blue: 0.0, alpha: 1.0001}.is_valid());
         assert!(!Color {red: 255.0001, green: 255.0001, blue: 255.0001, alpha: 1.0001}.is_valid());
     }
+
+    #[test]
+    fn ensure_value_mapping() {
+        let expected = Color { red: 65., green: 122., blue: 200., alpha: 1.};
+        let actual_rgb = Color::rgb(65., 122., 200.);
+        let actual_rgba = Color::rgba(65., 122., 200., 1.);
+        assert_eq!(expected, actual_rgb);
+        assert_eq!(expected, actual_rgba);
+    }
+
+    #[test]
+    fn ensure_hsl() {
+        let expected = Color { red: 148., green: 139., blue: 193., alpha: 1.};
+        let actual = Color::hsl(250., 0.3, 0.65);
+        assert_eq!(expected, actual);
+    }
+
+    #[test]
+    fn ensure_hsla() {
+        let expected = Color { red: 148., green: 139., blue: 193., alpha: 1.};
+        let actual = Color::hsla(250., 0.3, 0.65, 1.0);
+        assert_eq!(expected, actual);
+    }
+
+    #[test]
+    fn ensure_achromatic_hsl() {
+        let expected = Color { red: 26., green: 26., blue: 26., alpha: 1.};
+        let actual = Color::hsl(180., 0., 0.1);
+        assert_eq!(expected, actual);
+    }
```

Comment:
Thanks for taking the time to write tests! These are very helpful. The more robustly we test, the less likely we are to have a lot of bugs. :smile: 

Could you try to change the code in `rgb`, `rgba`, `hsl`, and `hsla` in some subtle ways to see if these tests catch potential mistakes? For example, try changing a `<` to `<=` or to `>` and see if the tests fail. You can also try changing `+` to `-` or the other way around. You don't have to try and exhaustively hit every case, but it is good to do this to see how much your tests will actually catch. This kind of "testing your tests" ensures that these tests are actually solid against potential future mistakes.